src/components/LogStream/LogLevelSummaryChart.tsx

```diff
@@ -36,8 +36,8 @@ function LogLevelSummaryChart({
     <Card className="h-100">
       <Card.Header className="p-2 px-3">
         <Row>
-          <Col>Log Occurences by Level</Col>
-          <Col className="text-right">
+          <Col data-name="logOccurenceTitle">Log Occurences by Level</Col>
```

Comment:
If I understand that correctly data-name refers to the name of the actual element? If yes, wouldn't element-name be more appropiate? Also what do you think about making the naming more streamlined by calling it dataName/elementName?

Reply:
Yes, `data-name` is name of the element we want to select from test.

I don't think `element-name` is an HTML5 attribute.

`data-*` is the only one I know of.

>  data-* | Global attribute | Lets you attach custom attributes to an HTML element.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

Using `data-*` is as per best practices of cypress https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

They suggest to use `data-cy`/`data-test`/`data-testid`. I thought `data-name` is more readable. We can definitely change this if we want to.

Reply:
Alright, thanks for the clarification!